docs/index.md

```diff
@@ -2,15 +2,20 @@
 
 
 ### Table of Contents
-* [Introduction](#introduction)
-* [Test Stratagies and Methodology](#test-strategies-and-methodology)
-* [Best Practices](#best-practices)
-* [Tooling](#tooling)
-  * [Workflow](#workflow)
-  * [Cluster recovery checks, metrics evaluation and pass/fail criteria](#cluster-recovery-checks-metrics-evaluation-and-passfail-criteria)
-* [Scenarios](#scenarios)
-* [Test Environment Recommendations - how and where to run chaos tests](#test-environment-recommendations---how-and-where-to-run-chaos-tests)
-* [Chaos testing in Practice within the OpenShift Organization](#chaos-testing-in-practice-within-the-OpenShift-Organization)
+- [Chaos Testing Guide](#chaos-testing-guide)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Test Strategies and Methodology](#test-strategies-and-methodology)
+  - [Best Practices](#best-practices)
+  - [Tooling](#tooling)
+    - [Workflow](#workflow)
+    - [Cluster recovery checks, metrics evaluation and pass/fail criteria](#cluster-recovery-checks-metrics-evaluation-and-passfail-criteria)
+  - [Scenarios](#scenarios)
+  - [Test Environment Recommendations - how and where to run chaos tests](#test-environment-recommendations---how-and-where-to-run-chaos-tests)
+    - [Chaos testing in Practice within the OpenShift Organization](#chaos-testing-in-practice-within-the-openshift-organization)
```

Comment:
Chaos testing in Practice is a new section, so we would need to keep it outside of the test environment recommendations. Thoughts?

Reply:
i have added it to the main index because no part dedicated was found. I don't see it under a test environement, but if you need to move it into another secction, tell mee where to place it.

Reply:
@startx-lab So basically we need to keep it the same way as it was before as a different section after "Test Environment Recommendations - how and where to run chaos tests": https://github.com/chaos-kubox/krkn/blob/main/docs/index.md.